### PR TITLE
[NOJIRA] Initial release

### PR DESCRIPTION
**Pull request by imrekel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,8 +29,8 @@ You can also run this step directly with [Bitrise CLI](https://github.com/bitris
 
 | Key | Description | Flags | Default |
 | --- | --- | --- | --- |
-| `project_root_path` | Path to the root folder of the project to be built | required | `.` |
-| `derived_data_path` | Path to the DerivedData folder used by the build. This must be the same folder specified for the -derivedDataPath flag when running xcodebuild. | required | `$HOME/Library/Developer/Xcode/DerivedData` |
+| `project_root_path` | The key used to store the cache archive | required | `.` |
```
Comment (zsolt-marta-bitrise) [inline README.md:32]: I think these 2 are mixed up

File: step.sh [hunk 2]
```diff
@@ -28,12 +28,16 @@ echo "Bitrise Build Cache is activated in this workspace"
 set -x
 
 # download the Bitrise Build Cache CLI
-export BITRISE_BUILD_CACHE_CLI_VERSION="v0.15.0-alpha.8"
+export BITRISE_BUILD_CACHE_CLI_VERSION=${BITRISE_BUILD_CACHE_CLI_VERSION:="v0.15.0-alpha.12"}
```
Comment (zsolt-marta-bitrise) [inline step.sh:31]: Is this going to be an alpha release?

File: e2e/bitrise.yml [hunk 3]
```diff
@@ -31,8 +33,13 @@ workflows:
             set -ex
 
             rm -rf $HOME/Library/Developer/Xcode/DerivedData
+    - git::https://github.com/bitrise-steplib/bitrise-step-restore-xcodebuild-cache.git@prepare-release:
```
Comment (zsolt-marta-bitrise) [inline e2e/bitrise.yml:36]: Shouldn't we also check the existence of directories?

File: step.sh (hunk 2 above)
Comment (imrekel) [reply step.sh:31]: we can switch to a non-alpha when the CLI is merged into master (after perhaps adding some more tests) - but I think it's ok to leave it like this until we give it to actual customers

File: e2e/bitrise.yml (hunk 3 above)
Comment (imrekel) [reply e2e/bitrise.yml:36]: done

File: e2e/bitrise.yml [hunk 4]
```diff
@@ -31,8 +33,41 @@ workflows:
...
+              echo "DerivedData folder does not exist"
+              exit 1
+            fi
+
+            metadata_path="dd-metadata.json"
+
+            if [ -f "$metadata_path" ]; then
+              echo "Cache metadata file exists"
+            else
+              echo "Cache metadata file does not exist"
```
Comment (zsolt-marta-bitrise) [inline e2e/bitrise.yml:68]: Shouldn't we `exit 1` here too?

Comment (imrekel) [reply e2e/bitrise.yml:68]: good catch

**Review by zsolt-marta-bitrise — APPROVED** [review]

**Pull request merged**